### PR TITLE
Update 12-bookdown.Rmd

### PR DESCRIPTION
**Pull request by mikey-harper**
# Chapter Outline
The aim of this is to capture my overall thoughts and propose an outline for the chapter layout.

### Shortening Process
The general aim is to condense the bookdown book (119 pages) into a single chapter within the RMarkdown book. Basic savings can be made:

-	Markdown syntax (25 pages): this is already covered in Chapter 
-	Appendices (16 pages)

This brings down the number of pages to around 78 page. About 10% of the content can be kept to bring the work down to the required length. How is it best to achieve this reduction and convert the book?

-  Reduce the number of concepts explained?
-  Reduce the detail of concepts explained?

I feel there will clearly be a balance between these two approaches to reduce the amount of content included (as proposed in outline).

### Relationship to bookdown book

Is this RMarkdown book aiming to replace or supplement the bookdown book? If it is aiming to supplement, can cross-references to the bookdown book be made? For example:

>	For more details on the customisation of the `_bookdown.yml`, see chapter X for extra details.

I feel it should be a guidepost, aiming to provide the reader with enough knowledge to get up and running, but also informing them of where to find out more information to achieve the more complex aspects.

### Differences in scope

The bookdown book was presented as a standalone document. Being a chapter of the full book, there is an opportunity for the writing to a new audience who had no necessarily even used Rmarkdown before.

Being part of the book, this chapter serves a slightly different purpose. The reader will know about Rmarkdown, but may need to understand the benefits that bookdown provide for long-form documents and some of the enhancements that it brings for bigger documents.

### CODE REVIEW

**Comment by mikey-harper** [comment]
@yihui , first pull request submitted so apologies if anything is not as it should be.

Happy to proceed with writing the full chapter this week but let me know your feedback first. Frantically reading your previous books in an attempt to learn your writing style and do my best to assimilate it.

File: 12-bookdown.Rmd [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+	-	Special headers
+	-	Text references
+	-	Cross-references
+
+## Output formats:
+
+- Highlight multiple output formats easily developed for same document.
+- Shorten description, but keep list of the YAML options for each format.
+
+## Customisation
```
Comment (yihui) [inline 12-bookdown.Rmd]: Actually I'm okay if this section is left out in this chapter.

**Comment by mikey-harper** [comment]
I agree with your changes, providing they get mentioned elsewhere in the book.

Things are coming together, and a draft should be finished by Sunday.

**Comment by yihui** [comment]
Sounds good!

**Pull request closed without merging**